src/lib.rs

```diff
@@ -1073,31 +1089,63 @@ impl AssociatedType {
     }
 }
 
+// ===== impl Field =====
+
+impl Field {
+    /// Return a field definition with the provided name and type
+    pub fn new<T>(name: &str, ty: T) -> Self
+    where T: Into<Type>,
+    {
+        Field {
+            name: name.into(),
+            ty: ty.into(),
+            documentation: Vec::new(),
+            annotation: Vec::new(),
+        }
+    }
+
+    /// Set field's documentation.
+    pub fn doc(&mut self, documentation: Vec<&str>) -> &mut Self {
+        self.documentation = documentation.iter().map(|doc| doc.to_string()).collect();
+        self
+    }
+
+    /// Set field's annotation.
+    pub fn annotation(&mut self, annotation: Vec<&str>) -> &mut Self {
```

Comment:
Maybe ```annotation(&mut self, annotation: &str)``` like ```import(&mut self, path: &str, ty: &str)```?

Reply:
I'm not sure I follow. Do you mean using a `&str` instead of a `Vec<&str>` for the `annotation()` method?

It was changed to use a `Vec` in https://github.com/carllerche/codegen/pull/6/commits/6e17dc251c40eb8729b819287fc533c68ff60030 to support multiple lines.

Reply:
I meant a method to add one annotation line per call, like you do for import(...) - each call results in a new import - instead of passing the whole Vec with all lines :)